### ISSUE TITLE
Missing documentation 

### ISSUE THREAD

**Issue by demihumandev** [fian46/addons-btree#52]
It would be really helpful. 

The basic nodes like selector, sequence, task and inverter are self explanatory but what about the other nodes? 
The range, count, while, priority selector and priority condition nodes are not obvious. Count node, does it count for ticks or seconds? Does while only return success of failed? Why does priority need a condition and how is it different from using a while node?

**Comment by fian46** [comment]
Count is for ticks

**Comment by fian46** [comment]
Priority will select top most node to run each tick

**Comment by Ezenki** [comment]
yes please any form of documentation on what each nodes do would be awesome, aside from that, this is a great plugin
